docs/css-classes-override.md

```diff
@@ -0,0 +1,65 @@
+# Customizing the Player CSS
+The player user interface (UI) is created using HTML and CSS, which mean you can edit the UI according to your needs, such as changing the color of the scrubber, changing the text font family, changing the control icons, and much more.
+
+This [demo](https://codepen.io/presentation_k/pen/XVQEXZ?editors=1100) is an example of how you can customzie the UI using the CSS.
+
+And here are some of the classes you can override:
+![Here is some of the classes you can override](./images/cssOverride-nocopy.png)
+
+
+
+> **Important!**
+> - The namespace for the default skin must be 'playkit'.
+> - The player uses the same font family in all of its components. You can override it in the general parent class (`.playkit-player-gui`) or customize each component according to your preferences.
+
+
+## Player Classes List
+
+Here's a list of the player classes you can customize:
+
+**Buttons**
+
+|Class Name| Description |
+|--|--|
+| .`playkit-pre-playback-play-button` | The background of the play button, before the player starts playing |
+| `.playkit-icon-play` | The player icon, which is used in the bottom controllers and on the player itself |
+|`.playkit-icon-start-over`| Start over icon, shown at the end of the video|
+| `.playkit-control-button` | The same for all the controllers icon buttons in the bottom control |
```

Comment:
I don't understand what you mean by "The same for all the controllers icon buttons in the bottom control" - please clarify!

Reply:
"a class attached to all the buttons in the control container, a change here will affect all the buttons"

Reply:
So let's change this to "This is a class that is attached to all the buttons in the control container; a change here will affect all of the buttons."

OK